partner_invoicing_mode_at_shipping/migrations/16.0.1.1.0/pre-migrate.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+# pylint: disable=odoo-addons-relative-import
+from odoo.addons.partner_invoicing_mode_at_shipping.hooks import (
+    _add_one_invoice_per_shipping,
+)
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    _add_one_invoice_per_shipping(env)
```

Comment:
@StefanRijnhart IMO it's OK to keep this migration script as it seems idempotent without bringing issues even on a code base > to 16.0. But for further migrations, to avoid their removal, we should at least let a comment there that we keep this migration script on purpose.